server/src/main/java/org/prlprg/session/DESCRIPTION.java

```diff
@@ -0,0 +1,90 @@
+
+package org.prlprg.session;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/** DESCRIPTION file parser. */
+public class DESCRIPTION {
+    private String version;
+    private final List<String> suggests;
+    private final List<String> imports;
+
+    public DESCRIPTION(Path descriptionFile) throws IOException {
+        this.suggests = new ArrayList<>();
+        this.imports = new ArrayList<>();
+        parseDescriptionFile(descriptionFile);
+    }
+
+    /**
+     * Parses the DESCRIPTION file.
+     *
+     * @param descriptionFile the path to the DESCRIPTION file
+     * @throws IOException if an I/O error occurs
+     */
+    private void parseDescriptionFile(Path descriptionFile) throws IOException {
+        List<String> lines = Files.readAllLines(descriptionFile);
+        Pattern versionPattern = Pattern.compile("^Version:\\s*(.*)$");
+        Pattern suggestsPattern = Pattern.compile("^Suggests:\\s*(.*)$");
+        Pattern importsPattern = Pattern.compile("^Imports:\\s*(.*)$");
```

Comment:
You can those as `static final` members of the class.
They will be compiled only once instead of each time that method is called.